docs/sources/installation/upgrading.md

```diff
@@ -327,6 +327,16 @@ For example, if you want an alert to be `INFO`-level in VictorOps, create a tag
 
 Grafana now requires all plugins to be signed. If a plugin is not signed Grafana will not load/start it. This is an additional security measure to make sure plugin files and binaries haven't been tampered with. All Grafana Labs authored plugins, including Enterprise plugins, are now signed. It's possible to allow unsigned plugins using a configuration setting, but is something we strongly advise against doing. For more information about this setting, refer to [allow loading unsigned plugins]({{< relref "../administration/#allow_loading_unsigned_plugins" >}}).
 
+### Grafana Live
+
+Grafana now maintains persistent WebSocket connections for real-time messaging needs.
+
+During WebSocket connection establishment Grafana checks request Origin header due to security reasons (to prevent hijacking of WebSocket connection). If you have a properly defined public URL (`root_url` server option) then origin check should successfully pass for WebSocket requests originating from public URL pages. In case of unsuccessful origin check Grafana returns 403. It's also possible to add a list of additional origin patterns for the origin check.
+
+To handle many concurrent WebSocket connections you may need to tune your OS settings or infrastructure. Grafana Live enabled by default and supports 100 concurrent WebSocket connections max to avoid possible problems with file descriptor OS limit. As soon as your setup meets requirements to scale number of persistent connections this limit can be increased. You also have an option to disable Grafana Live.
```

Comment:
```suggestion
To handle many concurrent WebSocket connections you may need to tune your OS settings or infrastructure. Grafana Live is enabled by default and supports 100 concurrent WebSocket connections max to avoid possible problems with the file descriptor OS limit. As soon as your setup meets the requirements to scale the number of persistent connections this limit can be increased. You also have an option to disable Grafana Live.
```